docs/get-started.rst

```diff
@@ -182,6 +182,25 @@ Then install the package using the default configuration:
 The server should now be running on http://localhost:8888
 
 
+Upgrading Kinto
+---------------
+
+Whenever you need to upgrade Kinto, first make sure that the server is stopped,
+and that the virtualenv is activated:
+
+::
+
+    source env/bin/activate
+
+Now upgrade Kinto (and its dependencies) using the following command:
+
+::
+
+    pip install --upgrade kinto
```

Comment:
Yes it does because [Cliquet's version isn't pinned](https://github.com/Kinto/kinto/blob/master/setup.py#L21). I've double checked by running this command, and it does indeed upgrade all the dependencies that don't have pinned versions.